### DISCUSSION TITLE
Problema com CORS

### DISCUSSION THREAD

**Discussion by MayconRDO** [balta-io/forum#242] [Tira Dúvidas] [answered]
Boa tarde @andrebaltieri , tudo bem?

Como faço para resolver o problema do CORS?

![image](https://user-images.githubusercontent.com/41632489/215183861-7554c128-2b9f-45e0-a294-1df182955910.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @MayconRDO, veja se te ajuda:
https://learn.microsoft.com/pt-br/aspnet/core/security/cors?view=aspnetcore-7.0
